src/migrations/1677778846950_lists-and-picks.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from "node-pg-migrate"
+
+const PICKS_TABLE = "picks"
+const LISTS_TABLE = "lists"
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable(LISTS_TABLE, {
+    id: { type: "uuid", notNull: true, primaryKey: true, unique: true, default: pgm.func("uuid_generate_v4()") },
+    name: { type: "text", notNull: true },
+    description: { type: "text", notNull: false },
+    user_address: { type: "text", notNull: true },
+  })
+
+  // Should create an index by those two columns
+  pgm.addConstraint(LISTS_TABLE, "name_user_address_unique", {
+    unique: ["name", "user_address"],
```

Comment:
I think that it wouldn't hurt to have multiple lists with the same name, specially as the lists are unique to each user. What do you think about this?
```suggestion
    unique: ["user_address"],
```

Reply:
What I tried to write here was a unique constraint that doesn't allow a user to have the same list name twice. That's why I wrote this specific constraint instead of using the definition of the `user_address` column.

Did I put something wrong?